fdbserver/DDTxnProcessor.actor.cpp

```diff
@@ -23,8 +23,25 @@
 #include "fdbclient/ManagementAPI.actor.h"
 #include "fdbserver/DataDistribution.actor.h"
 #include "fdbclient/DatabaseContext.h"
+#include "flow/genericactors.actor.h"
 #include "flow/actorcompiler.h" // This must be the last #include.
 
+static void updateServersAndCompleteSources(std::set<UID>& servers,
```

Comment:
This function is used by both `DDTxnProcessor` and `DDMockTxnProcessor` implementation